Kitodo-API/src/main/java/org/kitodo/api/validation/longtermpreservation/LtpValidationConditionInterface.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.validation.longtermpreservation;
+
+import java.util.List;
+
+/**
+ * Describes a condition that has to be met when validating files for long term preservation (ltp).
+ * 
+ * <p>For example, a validation condition for a tif file might be that the file is stored in
+ * one of several acceptable color spaces (RGB, CMYK).</p>
+ * 
+ * <p>A condition is modelled by the simple format {property} {operation} {value(s)}, e.g., 
+ * {ColorSpace} {contained in} {RGB,CMYK}.</p>
```

Comment:
This struck me as difficult to read at first glance. It's not a mistake, but I suggest an improvement:

```suggestion
 * <p>
 * A condition is modelled by the simple format "{property} {operation} {value(s)}".
 * Example: "{ColorSpace} {contained in} {RGB,CMYK}".
```